packages/shared/src/react/contexts.tsx

```diff
@@ -50,6 +50,23 @@ const OrganizationProvider = ({
   );
 };
 
+function useAssertWrappedByClerkProvider(displayNameOrCustomHandler?: string | (() => void)): void {
+  const ctx = React.useContext(ClerkInstanceContext);
+
+  if (!ctx) {
+    if (typeof displayNameOrCustomHandler === 'function') {
+      displayNameOrCustomHandler();
+      return;
+    }
+
+    throw new Error(
+      `${
+        displayNameOrCustomHandler || 'Clerk components'
+      } must be wrapped within the <ClerkProvider /> component. Learn more: https://clerk.com/docs/components/clerk-provider`,
+    );
+  }
```

Comment:
🔧 i would expect to have a displayName. Could we make the `displayName` param to be required and remove the `'Clerk components'` fallback?